attributes/default.rb

```diff
@@ -2,6 +2,9 @@
 default['packagecloud']['gpg_key_path'] = "/gpg.key"
 default['packagecloud']['hostname'] = nil
 
+default['packagecloud']['_stored_hostname'] = nil
+default['packagecloud']['override_hostname'] = nil
```

Comment:
I think the presence of now three attributes, all of which are a variant of hostname, presents a usability problem.